### DISCUSSION TITLE
Databases of legal torrents

### DISCUSSION THREAD

**Discussion by josecelano** [torrust/torrust-index-gui#455] [Show and tell]
I'm just collecting links. This might be a repo in the future.

**Comment by josecelano** [comment]
### Internet Archive

- https://blog.archive.org/2012/08/07/over-1000000-torrents-of-downloadable-books-music-and-movies/
- https://archive.org/details/bittorrent
- https://archive.org/advancedsearch.php?q=format%3A%22Archive+BitTorrent%22&fl%5B%5D=identifier&sort%5B%5D=&sort%5B%5D=&sort%5B%5D=&rows=50&page=1&output=json&callback=callback&save=yes
- https://help.archive.org/help/archive-bittorrents/

**Reply by josecelano** [reply]
![image](https://github.com/torrust/torrust-index-gui/assets/58816/b96e6f0f-ce65-4ac8-8251-0b665de29747)

**Comment by josecelano** [comment]
### Public Domain Torrents

- https://www.publicdomaintorrents.info/

**Comment by josecelano** [comment]
### LibriVox

https://librivox.org/

**Comment by da2ce7** [comment]
I liked this site: https://www.legittorrents.org/
However we will need to contact the old hostmaster and get a backup maybe...
